.buildkite/pipeline-raw.yml

```diff
@@ -0,0 +1,161 @@
+# SPDX-FileCopyrightText: 2019-2021 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+env:
+  SET_VERSION: "export TEZOS_VERSION=\"$(cat nix/nix/sources.json | jq -r '.tezos.ref' | cut -d'/' -f3)\""
+
+steps:
+ - label: reuse lint
+   command: nix run -f . pkgs.reuse -c reuse lint
+ - label: check trailing whitespace
+   command: .buildkite/check-trailing-whitespace.sh
+ - label: crossref-verify
+   command: "nix run -f https://github.com/serokell/crossref-verifier/archive/68a1f9d25b6e7835fea8299b18a3e6c61dbb2a5c.tar.gz -c crossref-verify"
+   soft_fail: true
+
+ - label: build via nix
+   commands:
+   - nix-build ./nix -A binaries -o binaries
+   branches: "!master"
+ - label: test nix-built binaries
+   commands:
+   - nix-build tests/tezos-nix-binaries.nix --no-out-link
+   branches: "!master"
+
+ - label: build via docker
+   commands:
+   - eval "$SET_VERSION"
+   - cd docker
+   - ./docker-static-build.sh
+   - nix run -f.. pkgs.upx -c upx tezos-*
+   artifact_paths:
+     - ./docker/tezos-*
+   agents:
+     queue: "docker"
+   only_changes: &static_binaries_changes_regexes
+   - docker\/build/\.*
+   - docker\/docker-static-build\.sh
+   - nix\/nix\/sources\.json
+   - meta\.json
+   - protocols\.json
+ # arm builer is an ubuntu machine without nix
+ - label: build arm via docker
+   commands:
+   - eval "$SET_VERSION"
+   - cd docker
+   - ./docker-static-build.sh
+   - upx tezos-*
+   - >
+     for f in ./tezos-*; do
+       mv "\$f" "\$f-arm64"
+     done
+   artifact_paths:
+     - ./docker/tezos-*
+   agents:
+     queue: "arm64-build"
+   only_changes: *static_binaries_changes_regexes
+
+ - label: check brew formulas
+   commands:
+     # Check all formulas except 'tezos.rb' because it's a base class for all other formulas
+     - find ./Formula -type f -name "*.rb" -exec ruby -c {} +
+     # All formulas share the same source URL inherited by the Tezos class, so it's fine
+     # to fetch sources for only one formula
+     - brew fetch -s ./Formula/tezos-client.rb
+   agents:
+     queue: "x86_64-darwin"
+   only_changes:
+   - Formula\/.*\.rb
+
+ # binaries built by docker are tested on the another machine,
+ # so we should wait for docker build to finish
+ - wait
+ - label: test docker-built binaries
+   commands:
+   - buildkite-agent artifact download "docker/*" . --step "build via docker"
+   - chmod +x ./docker/*
+   - nix-build tests/tezos-binaries.nix --no-out-link --arg path-to-binaries ./docker
+   branches: "!master"
+   only_changes:
+   - docker\/build/\.*
+   - docker\/docker-static-build\.sh
+   - nix\/nix\/sources\.json
+   - meta\.json
+   - protocols\.json
+   - tests\/test_script\.py
+   - tests\/tezos-binaries\.nix
```

Comment:
There are also a lot of files from `./nix` referenced in `tests/tezos-binaries.nix`, I guess they have to be tracked too

Reply:
hmm, apparently, it's easier to add just `nix/.*` because static binaries building is usually cached and tests aren't that huge